routes/web.php

```diff
@@ -13,6 +15,17 @@
 |
 */
 
+Route::resource('book', 'BookController', [
+    'as' => 'prefix'
```

Comment:
Do you need a prefix for this?

Reply:
Normally not much but I learned it in the tutorial and kind a liked the idea.. that is the reason I left it in the code..